src/components/ItemFilter/FilterSettingsDialog.js

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { DIMENSION_ID_PERIOD, DIMENSION_ID_ORGUNIT } from '@dhis2/analytics'
+import i18n from '@dhis2/d2-i18n'
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+    Radio,
+    Transfer,
+} from '@dhis2/ui'
+
+import classes from './styles/FilterSettingsDialog.module.css'
+
+const RadioOptions = ({
+    filtersSelectable,
+    updateFilterDimensionRestrictability,
+}) => (
+    <form>
+        <div className={classes.radioButton}>
+            <Radio
+                dense
+                checked={!filtersSelectable}
+                label={i18n.t('Allow filtering on all dimensions')}
+                name="radio-allow-filtering-on-all"
+                onChange={payload => {
+                    return updateFilterDimensionRestrictability(
+                        payload.value === 'true'
+                    )
+                }}
+                value={'false'}
+            />
+        </div>
+        <div className={classes.radioButton}>
+            <Radio
+                dense
+                checked={filtersSelectable}
+                label={i18n.t('Only allow filtering on specified dimensions')}
```

Comment:
I suggest updating the wording to: `Only allow filtering by selected dimensions`

Reply:
fixed